cartoframes/data/observatory/category.py

```diff
@@ -18,12 +18,11 @@ def _constructor_expanddim(self):
         return Categories
 
     @staticmethod
-    def get_by_id(category_id):
-        return get_category_repo().get_by_id(category_id)
+    def by_id(category_id):
```

Comment:
I guess this is just `syntax sugar` but let's agree on this API design decision.

I liked the `get_*` version for a couple of reasons:

- In general I see functions as `verbs` (where reasonable)
- You have all the `get` functions under the same `umbrella`. It's easier if you use an auto-complete system to write `get` and see what different options you have to get `Categories`
- There are other frameworks (such as Django) that follow that convention `get_by_*`

The case for `countries()` and `categories()` is a bit different, those are known classes and since they can't just be properties I'm OK with keeping them as `nouns`

Thoughts? (not sure if we changed it for any other reason)

Reply:
I like those reasons. I changed back all the methods names :+1: